internal/controllers/scheduling/controller.go

```diff
@@ -138,15 +138,14 @@ func (c *controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	if err := c.dispatchOp(ctx, op); err != nil {
 		if errors.IsInvalid(err) {
-			logger.V(0).Info("conflict while dispatching synthesis")
-			return ctrl.Result{}, nil
+			return ctrl.Result{}, fmt.Errorf("conflict while dispatching synthesis")
```

Comment:
It shouldn't be possible that we need to requeue on conflict, but if there is any kind of trivial bug in the `op` logic it becomes more likely. So it's safer to just requeue to mitigate the risk.